bridge_node:
   ros__parameters:
      id: "raisin_bridge"
      network_type: 0 # 0 for TCP, 1 for WEBSOCKET
      network_interface: "" # for TCP
      peer_id: "railab_raibo" # for TCP
      peer_ip: "192.168.12.1" # for WEBSOCKET
      topics_ros2_to_raisin: # format should be the vector of "$type, $name"
         ["std_msgs/msg/String, string1",
         "std_msgs/msg/String, string2"]
      topics_raisin_to_ros2: # format should be the vector of "$type, $name"
         ["std_msgs/msg/String, string3",
         "std_msgs/msg/String, string4"]